gamemodes/terrortown/entities/weapons/weapon_ttt_binoculars.lua

```diff
@@ -61,9 +61,9 @@ SWEP.ProcessingDelay = 5
 ---
 -- @ignore
 function SWEP:SetupDataTables()
-	self:DTVar("Bool", 0, "processing")
-	self:DTVar("Float", 0, "start_time")
-	self:DTVar("Int", 1, "zoom")
```

Comment:
Yes, why would we use the 2nd slot instead of the first? shouldn't matter too much (unless something sets the first integer slot on this entity. Which should not happen)